src/MasterBlocks.php

```diff
@@ -173,6 +173,7 @@ public function enqueue_block_public_assets(): void
     private function get_p4_options(): array
     {
         $option_values = get_option('planet4_options');
+        $is_new_ia = $option_values['new_ia'] === 'on' ;
```

Comment:
I see that in other places in the code we use [this check](https://github.com/greenpeace/planet4-master-theme/blob/main/src/Settings.php#L62), maybe we should be consistent and use this here too?